src/vorta/utils.py

```diff
@@ -20,6 +20,10 @@
 from vorta.log import logger
 from vorta.network_status.abc import NetworkStatusMonitor
 
+# Used to store whether a user wanted to override the
+# default directory for the --development flag
+DEFAULT_DIR_FLAG = "USE_DEFAULT_DIR"
```

Comment:
```suggestion
DEFAULT_DIR_FLAG = object()
```

This is an actually unique value. The user could theoretically supply the path `USE_DEFAULT_DIR`.

Reply:
Ah, that's clever. I wasn't sure how to go about this and just went for a string that was unlikely to be used but still somewhat informative.